packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -183,22 +183,20 @@ export class TreeItem extends FASTElement {
                 break;
             case keyCodeArrowDown:
                 // preventDefault to ensure we don't scroll the page
-                e.preventDefault();
                 this.focusNextNode(1);
                 break;
             case keyCodeArrowUp:
                 // preventDefault to ensure we don't scroll the page
```

Comment:
If we remove prevent default, should we also remove the comment for it in the line above?

Reply:
I left it in so as to not remove the information that it in fact prevents default (or returns true now) to prevent the page from scrolling. Do you think probably modifying it might be a better option?

Reply:
Updated the comment :D